weave/run_tests.py

```
@@ -0,0 +1,79 @@
+    runner = unittest.TextTestRunner(verbosity=2)
+    result = runner.run(suite)
+    
+    # Return exit code based on test results
+    return 0 if result.wasSuccessful() else 1
+
+def run_mcp_tests_only():
+    """Run only MCP-related tests"""
+    os.environ['WEAVE_TEST_MODE'] = 'true'
+    
+    # Import and run MCP tests specifically
+    from tests.test_mcp_management import TestMCPConfig, TestMCPSync, TestMCPIntegration
```

Pylint (3.3.7)

[error] No name 'test_mcp_management' in module 'tests'

(E0611)